crates/serpen/tests/fixtures/ast_rewriting/alias_transformation_test/main.py

```
@@ -0,0 +1,64 @@
+# These regular import aliases should be transformed/removed in bundled output
+import json as j
+import os as operating_system
+import sys as system_info
+
+# These from-import aliases should have their statements filtered
+from utils.data_processor import process_data as process_a
+from utils.data_processor import format_output as format_a
+from utils.config_manager import load_config as config_a
+
+# These non-aliased imports should remain unchanged
+import math
```

Pylint (3.3.7)

[convention] standard import "math" should be placed before third party imports "utils.data_processor.process_data", "utils.data_processor.format_output", "utils.config_manager.load_config"

(C0411)